### DISCUSSION TITLE
Keyboard shortcuts to sound volume

### DISCUSSION THREAD

**Discussion by dacomboman** [SpiralLinux/SpiralLinux-project#358] [General]
Can anyone help on this?

Much obliged!

p.s. still with SpiralLinux Cinnamon. ;)

**Comment by geckolinux** [comment]
Hi there, are you referring to the keyboard `Fn` + `something` keys for volume, or something else?

**Reply by dacomboman** [reply]
Do believe so.
I can get as far as "command" but where to after?

**Reply by geckolinux** [reply]
I'm sorry I don't understand, what are you trying to accomplish? What is currently working and what doesn't?

**Reply by dacomboman** [reply]
Found it!

Was right before my eyes!

Keyboard - shortcuts - sound and media - quiet keys 

Modified each one and they work perfectly!

Sorry for the bother. 😔
